### PR TITLE
Move stableswap subpool tests to new file

### PR DESCRIPTION
**Pull request by jepidoptera**
Also made some other changes, notably moving composite strategies into their designated file, and minor fixes and formatting corrections.

### CODE REVIEW

File: hydradx/model/amm/omnipool_amm.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def __init__(self,
         self.tvl_cap = tvl_cap
         self.stablecoin = preferred_stablecoin
         self.fail = ''
-        self.sub_pools = {}  # require sub_pools to be added through create_sub_pool
+        self.sub_pools = dict()  # require sub_pools to be added through create_sub_pool
```
Comment (poliwop) [inline hydradx/model/amm/omnipool_amm.py:70]: I'm confused why there are changes to omnipool_amm.py in a PR for reorganizing tests?

Comment (jepidoptera) [reply hydradx/model/amm/omnipool_amm.py:70]: That probably should not be in here, but the only difference it makes is cosmetic - for some reason with {}, PyCharm believes it is a dict of Oracles. With dict(), it (somehow) understands that we are talking about subpools. Either way it has no effect on functionality. For that reason I propose leaving it here.

Comment (poliwop) [reply hydradx/model/amm/omnipool_amm.py:70]: Okay, let's leave it.

File: hydradx/tests/test_omnipool_amm.py [hunk 2]
```diff
@@ -26,119 +22,6 @@
...
-                'LRNA': (
-                    asset_dict[base_token]['LRNA'] * sub_pool_instance.liquidity[tkn]
-                    / asset_dict[base_token]['liquidity']
-                    if 'LRNA' in asset_dict[base_token] else
-                    asset_dict[base_token]['LRNA_price'] * sub_pool_instance.liquidity[tkn]
-                )
-            } for tkn in sub_pool_instance.asset_list})
-            sub_pool_instances[name] = sub_pool_instance
-
-    spot_prices = {
```
[the remark this answers is not in the document]
Comment (jepidoptera) [reply hydradx/tests/test_omnipool_amm.py:111]: Yes, I think we don't need it.

File: hydradx/tests/test_omnipool_amm.py [hunk 3]
```diff
@@ -26,119 +22,6 @@
...
-        ''.join(random.choice('abcdefghijklmnopqrstuvwxyz') for _ in range(3)): {
-            'liquidity': draw(asset_quantity_strategy),
-            'LRNA': draw(asset_quantity_strategy)
-        } for _ in range(token_count - 2)
-    })
-    return return_dict
-
-
-@st.composite
-def assets_reasonable_config(draw, token_count: int = 0) -> dict:
```
[the remark this answers is not in the document]
Comment (jepidoptera) [reply hydradx/tests/test_omnipool_amm.py:53]: It's here:
https://github.com/galacticcouncil/HydraDX-simulations/blob/6a82e82e9f6d2da64180b7278183f6b10381a007/hydradx/tests/strategies_omnipool.py#L53

File: hydradx/tests/strategies_omnipool.py [hunk 4]
```diff
@@ -94,3 +98,111 @@ def omnipool_reasonable_config(
...
+def omnipool_config(
+        draw,
+        asset_dict=None,
+        token_count=0,
+        lrna_fee=None,
+        asset_fee=None,
+        tvl_cap_usd=0,
+        sub_pools: dict = None,
+        imbalance: float = None,
+        withdrawal_fee=True
```
[the remark this answers is not in the document]
Comment (jepidoptera) [reply hydradx/tests/strategies_omnipool.py:136]: This is how PyCharm likes it - no space around the equals sign in function declarations, *unless* you a specifying a type.

[the remark this answers is not in the document]
Comment (poliwop) [reply hydradx/tests/strategies_omnipool.py:136]: Ah. That's... interesting.

**Pull request merged**